### DISCUSSION TITLE
screen.debug() only display the lite web component, not able to see shadowroot

### DISCUSSION THREAD

**Discussion by muz3** [KonnorRogers/shadow-dom-testing-library#56] [General]
Hi I am looking for direction help. I wonder if this is because my custom web-component is not loaded by jest and RTL? Is there any example use of this library to experiment?

**Comment by KonnorRogers** [comment]
https://stackblitz.com/edit/vitejs-vite-6eumsn?file=src%2FApp.tsx,package.json,src%2FApp.css,src%2FApp.test.tsx,jest.config.js,__mocks__%2FfileMock.js,__mocks__%2FstyleMock.js,setupTests.js,src%2FMyTextarea.tsx

You should need to load your web component to get Jest + RTL to render it.
